### PR TITLE
chore(build): update babel deps, fix production build of docs and build on Windows

### PR DESCRIPTION
**Pull request by layershifter**
This PR:
- update all Babel deps
- fixes issues with production build of docs
- fixes with build of lib on Windows

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -1,5 +1,9 @@
-import { task, series, parallel } from 'gulp'
-import path from 'path'
+require('@babel/register')({
+  ignore: [/node_modules/, /docs\/dist/],
+})
+
+const { task, series, parallel } = require('gulp')
+const path = require('path')
```
Comment (layershifter) [inline gulpfile.js:6]: Fix for https://github.com/nozzle/react-static/issues/776

File: gulp/sh.js [hunk 2]
```diff
@@ -1,13 +1,32 @@
...
 
-  exec(cmd, options, cb)
+  child.on('close', (code) => {
+    if (code === 0) {
+      cb(stdoutData)
+    }
+
+    cb(new Error(`child process exited with code ${code}`))
+  })
 }
```
Comment (layershifter) [inline gulp/sh.js:30]: Picked from https://github.com/stardust-ui/react/blob/master/build/gulp/sh.ts

**Comment by layershifter** [comment]
A manually tested staging build, works fine :+1:

**Comment by levithomason** [comment]
Released in `semantic-ui-react@0.83.0`.

**Pull request merged**